gisserver/output/csv.py

```diff
@@ -111,6 +111,13 @@ def get_header(self, fields: list[XsdElement]) -> list[str]:
         names = []
         append = names.append
         for field in fields:
+
+            if field.is_geometry:
+                append(field.name)
```

Comment:
vanwaar deze?

Reply:
Dat was een implementatie vraagje, ik heb de indruk dat je altijd een geometry veld wil meegeven in de output. Wilde ik even checken met Edward.